cpp/src/generated/parquet_types.cpp

```diff
@@ -10,6 +10,7 @@
 #include <ostream>
 
 #include <thrift/TToString.h>
+#include <thrift/transport/TBufferTransports.h>
```

Comment:
I realize that this is a generated file and we don't want to make changes here. However, I am unsure how this is compiled and/or how to make changes at the generation level to do a "minimal" read.